### ISSUE TITLE
Ubuntu Gnome bug

### ISSUE THREAD

**Issue by myrkath** [nikp123/xava#61]
I use XAVA on Linux Mint 22/ GNOME Wayland and am generally very happy with the program. However, I have one specific problem that I have not been able to solve. It concerns window positioning in the Wayland environment. While all other features work as expected, it seems that the options for adjusting window position are not implemented correctly(for gl).

Linux Mint 22
GNOME Shell 46.0/ Wayland
newest XAVA Appimage

**Comment by nikp123** [comment]
Yeah that's an issue with Wayland in general (no unified positioning protocol exists IIRC), since I don't really have control over that. Thankfully since I switched to Gnome I can probably take a shot at it.

**Comment by nikp123** [comment]
@myrkath do you mind checking if wayland windows actually work under Gnome. I tried testing those myself and they straight up are broken on the current ``master`` branch.

**Comment by myrkath** [comment]
I have several applications running under Wayland, and none of them are causing any issues. All applications are working flawlessly, and I haven't encountered any problems with the windows or overall performance.

**Comment by myrkath** [comment]
And the performance is better than in x11

**Comment by nikp123** [comment]
> I have several applications running under Wayland, and none of them are causing any issues. All applications are working flawlessly, and I haven't encountered any problems with the windows or overall performance.

Didn't mean Wayland itself, but the Wayland mode of XAVA. I had trouble lunching it

**Comment by myrkath** [comment]
Yea, the Xava backgound layer is very buggy in Wayland, sometimes Xava shows trough Windows and however sometimes it is over my mouse cursor.

**Comment by nikp123** [comment]
> Yea, the Xava backgound layer is very buggy in Wayland, sometimes Xava shows trough Windows and however sometimes it is over my mouse cursor.

Ok I think I fixed it with the latest commit. Although, be warned that it's quite barebones because there aren't any mouse or keyboard inputs implemented at the moment.

**Comment by myrkath** [comment]
Works much better now. Thank you
